src/config.json

```diff
@@ -251,7 +106,7 @@
           "input": {
             "defaultBasemapStyle": {
               "name": "Positron",
-              "styleFile": "${ARLAS_MAP_STYLE}"
+              "styleFile": "http://demo.arlas.io:82/styles/positron/style.json"
```

Comment:
I know it's for dev but  maybe you should add a comment to avoid forget it

Reply:
I will treat it in the next PR